.github/workflows/stage-release.yml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021-2024 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+name: "Stage release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      reviewer_1:
+        description: "The GitHub username of the first commits reviewer"
+        required: true
+      reviewer_2:
+        description: "The GitHub username of the second commits reviewer"
+        required: true
```

Comment:
So far, releases required only one reviewer.  When did this change?

Reply:
The script requires two reviewers if it's not execute by a committer.